src/constants.js

```diff
@@ -74,6 +74,9 @@ const SyncEvents = {
   }
 }
 
+// Settings
+const ACTIVE_CLASS_NAME = 'active'
+
```

Comment:
As this is only used in the `Settings.js` file, you might as well just specify it in there, saves you the comment and the import.